### PR TITLE
[13.0][MIG] website_sale_suggest_create_account: Migration to 13.0

### PR DESCRIPTION
**Pull request by Tardo**
cc @tecnativa TT21707

### CODE REVIEW

**Review by CarlosRoca13 — APPROVED** [review]
All is working well.

File: website_sale_suggest_create_account/hooks.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+
+def post_init_hook(cr, registry):
+    if config["test_enable"] or config["test_file"]:
+        with api.Environment.manage():
+            env = api.Environment(cr, SUPERUSER_ID, {})
+            env.ref("website_sale_suggest_create_account.cart").active = False
+            env.ref(
+                "website_sale_suggest_create_account.short_cart_summary"
+            ).active = False
```
Comment (sergio-teruel) [inline website_sale_suggest_create_account/hooks.py:22]: Why do you need this???

**Comment by pedrobaeza** [comment]
Please squash together translations coming from the same person.

**Review by sergio-teruel — APPROVED** [review]
OK.. only post_init_hook comment...

File: website_sale_suggest_create_account/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# Copyright (C) 2015 Antiun Ingeniería, S.L.
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
+{
+    "name": "Suggest to create user account when buying",
+    "summary": "Suggest users to create an account when buying in the website",
+    "version": "13.0.1.0.0",
+    "category": "Website",
+    "website": "https://www.tecnativa.com",
```
Comment (MeetKD) [inline website_sale_suggest_create_account/__manifest__.py]: website -> https://github.com/OCA/e-commerce/

File: website_sale_suggest_create_account/views/website_sale.xml [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+        </xpath>
+        <xpath expr="//a[@href='/shop/checkout?express=1']" position="after">
+            <a t-if="suggest_login" role="button"
+               class="btn btn-secondary float-right d-none d-xl-inline-block"
+               href="/web/login?redirect=/shop/checkout?express=1">
+                <span>Log in and checkout</span>
+            </a>
+            <t t-if="suggest_create_account">
+                <a class="btn btn-secondary float-right d-none d-xl-inline-block" role="button"
+                   href="/web/signup?redirect=/shop/checkout?express=1" style="margin-top:3px">
```
Comment (MeetKD) [inline website_sale_suggest_create_account/views/website_sale.xml]: Can we remove style here  and add margin top class ?

**Review by MeetKD — CHANGES_REQUESTED** [review]

File: website_sale_suggest_create_account/hooks.py (hunk 1 above)
Comment (Tardo) [reply website_sale_suggest_create_account/hooks.py:22]: Explained here: https://github.com/OCA/e-commerce/pull/336/files#diff-981e540460b2382c62143803bf7ae257R8

**Comment by Tardo** [comment]
Changes Done!

**Comment by Tardo** [comment]
ping @MeetKD

**Review by MeetKD — APPROVED** [review]

**Pull request merged**